Documentation/Usage.md

```diff
@@ -146,81 +146,106 @@ following files from the libgit2 system-package are of interest:
     /usr/local/lib/libgit2.dylib      # .so on Linux
     /usr/local/include/git2.h
 
-Swift packages that provide module maps for system libraries are handled
-differently from regular Swift packages.
+**Note:** the system library may be located elsewhere on your system, such as
+`/usr/`, or `/opt/homebrew/` if you're using Homebrew on an Apple silicon Mac.
+If you're not sure where `Clibgit` is in your system, you can run this command
+to look it up:
 
-Note that the system library may be located elsewhere on your system, such as
-`/usr/` rather than `/usr/local/`.
+    example$ pkg-config --cflags libgit2
+    -I/usr/local/libgit2/1.6.4/include
 
-Create a directory called `Clibgit` next to the `example` directory and
-initialize it as a package that builds a system module:
 
-    example$ cd ..
-    $ mkdir Clibgit
-    $ cd Clibgit
-    Clibgit$ swift package init --type empty
-
-This creates a `Package.swift` file in the directory.
-Edit `Package.swift` and add `pkgConfig` parameter:
+**First, let's define the `target` in the package description**:
 
 ```swift
-// swift-tools-version:5.3
+// swift-tools-version: 5.8
+// The swift-tools-version declares the minimum version of Swift required to build this package.
 
 import PackageDescription
 
 let package = Package(
-    name: "Clibgit",
-    pkgConfig: "libgit2"
+    name: "example",
+    targets: [
+        // systemLibrary is a special type of build target that wraps a system library
+        // in a target that other targets can require as their depencency.
+        .systemLibrary(
+            name: "Clibgit",
+            pkgConfig: "libgit2",
+            providers: [
+                .brew(["libgit2"]),
+                .apt(["libgit2-dev"])
+            ]
+        )
+    ]
 )
+
 ```
 
-The `pkgConfig` parameter helps SwiftPM in figuring out the include and library
-search paths for the system library. Note: If you don't want to use the `pkgConfig`
-parameter you can pass the path of a directory containing the library using the
-`-L` flag in commandline when building your app:
+**Note:** If you don't want to use the `pkgConfig` parameter you can pass the
+path of a directory containing the library using the `-L` flag in command line
+when building your app instead:
 
     example$ swift build -Xlinker -L/usr/local/lib/
 
-Create a `module.modulemap` file so it consists of the following:
+Next, we'll create a directory `Sources/Clibgit` in your `example` projcet, and
```

Comment:
Nit: typo, I'd also avoid the use of "we" here to stay consistent with using "you" in other paragraphs.
```suggestion
Next, create a directory `Sources/Clibgit` in your `example` project, and
```